martus-client/source/org/martus/client/test/TestSha1ToSha2Change.java

```diff
@@ -279,6 +280,36 @@ public void testParseRetrieveCommandBundleSignedWithSha2UsingSha1Verifier() thro
 		}
 	}
 
+	public void testExtractPublicInfoSignedWithSha1UsingSha2Verifier() throws Exception
+	{
+		File temp = createTempFile();
+		temp.delete();
+
+		appSha1.exportPublicInfo(temp);
+
+		String publicKey = appSha2.extractPublicInfo(temp);
+		assertEquals("Public Key wrong?", appSha2.getSecurity().getPublicKeyString(), publicKey);
+	}
+
+	public void testExtractPublicInfoSignedWithSha2UsingSha1Verifier() throws Exception
+	{
+		File temp = createTempFile();
+		temp.delete();
+
+		appSha2.exportPublicInfo(temp);
+
+		try
+		{
+			appSha1.extractPublicInfo(temp);
+			fail("testExtractPublicInfoSignedWithSha2UsingSha1Verifier should have thrown PublicInformationInvalidException");
+		}
+		catch (MartusUtilities.PublicInformationInvalidException e)
+		{
+			// expected exception
```

Comment:
Rename the "e" variable to "expectedException" and remove the comment //expected exception